components/automate-chef-io/content/docs/profiles.md

```diff
@@ -17,13 +17,13 @@ Navigate to compliance profiles by using the **Compliance** tab and then selecti
 Compliance profiles help you secure your infrastructure continuously.
 Chef Automate compliance profiles translate CIS Benchmarks and other security standards in easily readable policy.
 Chef Automate uses the InSpec language to turn compliance into code.
-You can install and download one of our 100+ ready-to-use compliance profiles from Profiles--or upload your own custom profiles.
+You can install and download one of our 300+ ready-to-use compliance profiles from Profiles--or upload your own custom profiles.
```

Comment:
Since we're looking at this section, we could consider revising it a bit. Ideally, the overview follows the "inverted pyramid" structure, which means that it starts with the general idea of what is a profile and why you would use them, and then it briefly introduces ours. 

I'm not sure how we've handled it on other pages, but I'm thinking that the Navigation instructions could be above the `overview` heading, so users see them distinctly.